### PR TITLE
Upgrade `aws-sdk` to version `3`

### LINKED ISSUE #244: Upgrade Ruby SDK to version `3`
Version 2 of the Ruby SDK will enter maintenance mode as of November 20, 2020. To continue receiving service updates and new features, please upgrade to Version 3. More information can be found here: https://aws.amazon.com/blogs/developer/deprecation-schedule-for-aws-sdk-for-ruby-v2/

### PR DESCRIPTION
**Pull request by waldofouche**
Followed the guide here: https://aws.amazon.com/blogs/developer/deprecation-schedule-for-aws-sdk-for-ruby-v2/
Resolves #244

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,4 @@ AAF Reporting Service generates reports and graphs from data sourced from:
 2. `brew install phantomjs`
 3. `bin/setup` (and read the output for any additional steps)
 4. `guard`
+5. `RAILS_ENV=test bundle exec rake db:reset` to run test suite
```
Comment (matthew-puku) [inline README.md:32]: either "`db:reset`" or "to run test suite" is incorrect here

Comment (waldofouche) [reply README.md:32]: ah hang on, let me check my terminal history

Comment (waldofouche) [reply README.md:32]: Ah okay yep, so I forgot to add `RAILS_ENV=test bundle exec rake`

File: README.md [hunk 2]
```diff
@@ -29,3 +29,7 @@ AAF Reporting Service generates reports and graphs from data sourced from:
 2. `brew install phantomjs`
 3. `bin/setup` (and read the output for any additional steps)
 4. `guard`
+
+## Running tests in development environment
+1. `RAILS_ENV=test bundle exec rake db:reset` to rest the database
```
Comment (matthew-puku) [inline README.md:34]:
```suggestion
1. `RAILS_ENV=test bundle exec rake db:reset` to reset the database
```

**Review by matthew-puku — APPROVED** [review]
LGTM

**Review by matthew-puku — APPROVED** [review]
i don't like the setting that dismisses reviews on push 😞

**Comment by waldofouche** [comment]
> i don't like the setting that dismisses reviews on push

Agreed!

**Pull request merged**